### PR TITLE
update stack transfer in docs

### PR DESCRIPTION
**Pull request by anita-trimbur**
small updates to our docs that mention stack transfer to align with the new bulk transfer feature.

fixes https://github.com/pulumi/pulumi-hugo/issues/2222

### CODE REVIEW

**Comment by susanev** [comment]
we should hold the merging of this until after the service release with changes goes out

File: themes/default/content/docs/support/faq.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ $ pulumi stack init acme-corp/widget-server
 
 The [Pulumi Service](https://app.pulumi.com/) allows you to transfer stacks from your individual account to any organization account you belong to as an administrator.
 
-To transfer a stack from your individual account to an organization, navigate to the project containing the stack you wish to migrate, then select the stack in question.
+To transfer a stack from your individual account to an organization, navigate to the Stacks page and select Transfer Stacks.
```
Comment (susanev) [inline themes/default/content/docs/support/faq.md:37]: i think our patten is to bold `Transfer stacks` also id drop the uppercase s to lowercase to align with the ui

**Review by susanev — APPROVED** [review]

File: themes/default/content/docs/support/faq.md (hunk 1 above)
Comment (anita-trimbur) [reply themes/default/content/docs/support/faq.md:37]: i saw that in the other pages, but the markdown looked totally different on this faq one. should i bold it anyway?

Comment (susanev) [reply themes/default/content/docs/support/faq.md:37]: yah you should be able to, its a .md file, surrounding it with `**` should do the trick

**Pull request merged**